ai-library/base/operator.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ailibrary-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: ailibrary-operator
+  template:
+    metadata:
+      labels:
+        name: ailibrary-operator
+    spec:
+      serviceAccountName: ailibrary-operator
+      containers:
+        - name: ansible
+          command:
+          - /usr/local/bin/ao-logs
+          - /tmp/ansible-operator/runner
+          - stdout
+          image: "quay.io/croberts/ai-library-operator"
+          imagePullPolicy: "Always"
```

Comment:
Can we place this image in the opendatahub quay account

Reply:
Yes, I'll make that happen.